LCLSGeneral/LCLSGeneral/POUs/Diagnostics/FB_EtherCATDiag.TcPOU

```diff
@@ -55,12 +55,15 @@ VAR
 
 	(* -- Check Topology *)
 	iDiagIndex					: ARRAY [0..ESC_MAX_PORTS] OF UINT;
+    rDiagIndex  :   REFERENCE TO UINT;    
```

Comment:
References should only really be used for structured data types - seems like this is more `iCurrentDiagIndex` or something?

Reply:
Yeah that felt awkward.